manager/controlapi/service_test.go

```diff
@@ -15,6 +15,20 @@ import (
 	"google.golang.org/grpc/codes"
 )
 
+func createCustomSpec(name, runtime string) *api.ServiceSpec {
+	spec := createSpec(name, runtime, 0)
+	spec.Task.Runtime = &api.TaskSpec_Custom{
+		Custom: &api.CustomSpec{
+			Name: runtime,
+			Payload: &gogotypes.Any{
+				TypeUrl: "com.docker.custom.runtime",
```

Comment:
It looks like we need something like `type.docker.com/docker.swarmkit.v1.PluginSpec` to be compliant but I am not sure where they have this specified.